rust/osm2lanes/src/transform/tags_to_lanes/mod.rs

```diff
@@ -117,22 +117,25 @@ impl std::convert::From<LaneBuilderError> for RoadError {
     }
 }
 
+#[derive(Debug, Clone, Copy)]
 enum LaneType {
     Travel,
     Parking,
     Shoulder,
 }
 
-#[derive(Default)]
+#[derive(Clone, Default, Debug)]
 struct Width {
     min: Infer<Metre>,
     target: Infer<Metre>,
     max: Infer<Metre>,
 }
 
-#[derive(Default)]
-struct LaneBuilder {
+#[derive(Clone, Default, Debug)]
+pub struct LaneBuilder {
     r#type: Infer<LaneType>,
+    // TODO: is it better to have this be the direction relative to the way, or to the side.
```

Comment:
The direction should be relative to the way, matching https://wiki.openstreetmap.org/wiki/Forward_%26_backward,_left_%26_right. If we did the alternative... the leftmost and rightmost lane could be reasoned about, but what about the middle lane on a 3 lane road? Meaningless to say it belongs to the left or right side of the road